### PR TITLE
adding the base files to create static docs site for webpack-dashboard

### PR DESCRIPTION
**Pull request by ebrillhart**
This adds all of the basic files that we can build off of to create a static docs site for the webpack-dashboard project.

/cc @paulathevalley

### CODE REVIEW

File: src/components/title-meta.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      title: this.props.title,
+      meta: {
+        property: {
+          "og:title": this.props.title
+        }
+      }
+    };
+
+    return (
+      <DocumentMeta {...titleMeta}>
```
Comment (paulathevalley) [inline src/components/title-meta.js]: :+1: I like this default addition!

File: src/components/title-meta.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import DocumentMeta from "react-document-meta";
+
+class TitleMeta extends React.Component {
```
Comment (paulathevalley) [inline src/components/title-meta.js:4]: :+1: Nice! I like this addition as part of the defaults.

Comment (ebrillhart) [reply src/components/title-meta.js:4]: it was in the component-playground-docs repo so I just copied it over! but glad it's a good addition 🎉

**Comment by paulathevalley** [comment]
:shipit:

**Pull request merged**